validator/params/config.go

```diff
@@ -7,6 +7,9 @@ import (
 	"math"
 )
 
+// SlotDuration in seconds.
+var SlotDuration = 8
```

Comment:
Why is this a stand alone value and not part of the config?

Reply:
Same as in the beacon-chain project. We have a params package where we define some globals to use, I just followed that same pattern.